lib/thrift/generator/struct_binary_protocol.ex

```diff
@@ -803,6 +771,90 @@ defmodule Thrift.Generator.StructBinaryProtocol do
     |> list_deserializer(name, file_group)
   end
 
+  def struct_serializer(%Union{fields: fields}, name, file_group) do
```

Comment:
Is there something like package visibility in Elixir? It's def (vs defp) because it's used by another module, but it's not something that end users should be coding against.